### PR TITLE
add documentation and some optimization

### PR DESCRIPTION
**Pull request by emjay0921**

### CODE REVIEW

File: spp_idpass/models/registrant.py [hunk 1]
```diff
@@ -108,111 +125,121 @@ def send_idpass_parameters(self, vals):  # noqa: C901
...
             else:
                 raise ValidationError(
                     _("ID PASS Error: %(reason)s Code: %(code)s")
                     % (response.reason, response.status_code)
-                )  # noqa: C901
-            _logger.info(
-                "ID PASS Response: %s Code: %s"
-                % (response.reason, response.status_code)
-            )
+                )  # noqa: C901\
```
Comment (jeremi) [inline spp_idpass/models/registrant.py]: the `\` is likely a mistake

**Review by jeremi — CHANGES_REQUESTED** [review]
Use the restructured format for the documentation. Also do not add a line that repeat the function name

File: spp_idpass/models/id_type.py [hunk 2]
```diff
@@ -23,6 +27,12 @@ def unlink(self):
                 return super(OpenG2PIDType, self).unlink()
 
     def write(self, vals):
+        """
+        This overrides the write function to stop default ID
+        Types from being edited
+        :param vals: The Values being edited.
+        :return: raise an error if the ID Type being edited is id_type_idpass else write.
```
Comment (jeremi) [inline spp_idpass/models/id_type.py]: Errors should not be in return, but in raises:
:raises: :class:`RuntimeError`: Out of fuel

File: spp_idpass/models/registrant.py [hunk 3]
```diff
@@ -37,19 +40,27 @@ def open_issue_idpass_wiz(self):
         }
 
     def send_idpass_parameters(self, vals):  # noqa: C901
+        """
+        These function is being used to handle the passing of Datas
+        to IDPass, to generate the ID of the registrant
```
Comment (jeremi) [inline spp_idpass/models/registrant.py]: IDPass -> ID PASS

File: spp_idpass/models/registrant.py [hunk 4]
```diff
@@ -37,19 +40,27 @@ def open_issue_idpass_wiz(self):
         }
 
     def send_idpass_parameters(self, vals):  # noqa: C901
+        """
+        These function is being used to handle the passing of Datas
```
Comment (jeremi) [inline spp_idpass/models/registrant.py]: These -> This

File: spp_idqueue/models/id_batch.py [hunk 5]
```diff
@@ -61,6 +61,9 @@ def refresh_data(self):
         return
 
     def approve_batch(self):
+        """
+        These are used to approve or validate a batch
```
Comment (jeremi) [inline spp_idqueue/models/id_batch.py]: These what?

**Review by jeremi — CHANGES_REQUESTED** [review]

File: spp_idqueue/models/id_batch.py (hunk 5 above)
Comment (emjay0921) [reply spp_idqueue/models/id_batch.py]: Done @jeremi

File: spp_idpass/models/registrant.py (hunk 4 above)
Comment (emjay0921) [reply spp_idpass/models/registrant.py]: Done @jeremi

File: spp_idpass/models/registrant.py (hunk 3 above)
Comment (emjay0921) [reply spp_idpass/models/registrant.py]: Done @jeremi

File: spp_idpass/models/id_type.py (hunk 2 above)
Comment (emjay0921) [reply spp_idpass/models/id_type.py]: Done @jeremi

File: spp_idpass/models/registrant.py (hunk 1 above)
Comment (emjay0921) [reply spp_idpass/models/registrant.py]: Done @jeremi

**Review by gonzalesedwin1123 — APPROVED** [review]

**Review by jeremi — APPROVED** [review]

**Pull request merged**